torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -31,6 +31,39 @@ const std::string kServerActiveAsyncCalls = "agent.server_active_async_calls";
 const std::string kRpcTimeoutErrorStr =
     "RPC ran for more than set timeout ({} ms) and will now be marked with an error";
 
+std::vector<c10::DeviceIndex> mapTensorDevices(
```

Comment:
Since this function doesn't actually * do * the mapping (i.e. move the tensors to device) would it be better named something like `getDevicesForTensor`?